scripts.d/s-salome-tar

```diff
@@ -133,6 +133,12 @@ function s-salome-tar-install()
         done
     fi
 
+    #  Do clean-up HELLO and PYHELLO for the release
+    for M in HELLO PYHELLO; do
+        sed -i -E "s&^.*SalomeAppConfig.*/BINARIES-DB10/$M/share/salome/resources/&#\0&g" \
```

Comment:
Hum... This is obviously wrong and will not work on DB11 or DB09. I would just remove this part, the match would still be unambiguous I think. Also please use `,` and not `&` as sed separator to be consistent with current sed commands.

So I propose:

`"s,^.*SalomeAppConfig.*/$M/share/salome/resources/,#\0,g"`

Please double check that it is working fine.